packages/typespec-vscode/README.md

```diff
@@ -19,11 +22,17 @@ Before using the TypeSpec extension, install [Node.js](https://nodejs.org/en/dow
 npm --version
 ```
 
+Install the TypeSpec CLI:
+
+```sh
+npm install -g @typespec/compiler
```

Comment:
this is not required

Reply:
I don't understand. Il is mentioned in the topic of installing TypeSpec https://typespec.io/docs/.  In fact, "create TypeSpec project" silently install the compiler locally to smooth further scenarios.  And "to preview API documentation" requires compiler/CLI.  User will encounter errors when there is no compiler installed either locally or globally.  Then why should this document not mention the compiler installation?

Reply:
The extension shouldn't even use the global version. Every time it does it produce an error.

The global version is just a way to init and give a global access to the tsp command .

In every other used case it should expect a local install of the compiler.

Everything should be able to work without this installed is my point.

Reply:
and poor experience also include https://github.com/microsoft/typespec/issues/6475